boards/arm/phy62xx/phy6222/src/bringup.c

```diff
@@ -70,7 +70,7 @@ extern int phyplus_timer_initialize(const char *devpath, int timer);
 
 int phy62xx_bringup(void)
 {
-  int ret;
+  int ret = OK;
```

Comment:
Let's remove `UNUSED(ret);` and update return to `return ret;`

Reply:
Done!